app/ts/utils.ts

```diff
@@ -109,18 +109,22 @@ export const getAccounts = async () => {
 	return reply[0]
 }
 
-const createClient = (account: AccountAddress) => {
+const createReadClient = (account: AccountAddress | undefined) => {
+	if (window.ethereum === undefined || account === undefined) {
+		return createPublicClient({ chain: mainnet, transport: http('https://geth.dark.florist') })
```

Comment:
In combination with publishing this change https://github.com/DarkFlorist/operations/pull/18, we should point this to a more generic endpoint.
```suggestion
		return createPublicClient({ chain: mainnet, transport: http('https://ethereum.dark.florist') })
```